README.md

```diff
@@ -195,6 +195,39 @@ npm run package --publish=never
 
 The build output goes to `dist/`.
 
+### Debugging builds with the Chrome remote debugger
+
+The Chrome remote debugger allows you to debug Electron apps using Chrome Developer Tools. You can use it to access log messages that might output to the developer console of the renderer process. This is especially helpful for getting additional debug information in production builds of Rancher Desktop.
```

Comment:
Even though it's not difficult to find out the folder where Rancher Desktop is installed, it helps if we provide that information here.

Reply:
I updated the PowerShell example to change the directory to the Rancher Desktop install location before starting with remote debugging enabled.